datatypes/src/primitives/time_interval.rs

```diff
@@ -35,8 +37,8 @@ impl Default for TimeInterval {
     /// ```
     fn default() -> Self {
         Self {
-            start: i64::min_value().into(),
-            end: i64::max_value().into(),
+            start: chrono::MIN_DATETIME.into(),
```

Comment:
can we use the min max from TimeInstancE?

Reply:
5a4d480